go/service/gregor.go

```diff
@@ -1691,12 +1691,6 @@ func (g *gregorHandler) DismissCategory(ctx context.Context, category gregor1.Ca
 		Ranges_: []gregor1.MsgRange{
 			gregor1.MsgRange{
 				Category_: category,
-				// A small non-zero offset that effectively means "now",
```

Comment:
@maxtaco of note this section, I think it is safe to remove this now with the server changes.

Reply:
👍